.github/workflows/node.js.yml

```
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2
       with:
-        node-version: 16.x
+        node-version: 18.x
```

The `.x` does not seem to be necessary (see [here](https://github.com/actions/setup-node#supported-version-syntax)):

```suggestion
        node-version: 18
```